main.js

```diff
@@ -1,27 +1,36 @@
-var $car = document.querySelector('.car')
+var $car = document.querySelector('.car');
 var go = null;
+var numberOfSpacePresses = 0;
 
 var data = {
   direction: 'east',
   location: {
     x: 20,
     y: 20
-  }
-}
+  },
+  moving: false
+};
 
-document.addEventListener('keydown', turnCar)
+document.addEventListener('keydown', turnCar);
 document.addEventListener('keydown', function (e) {
   if (e.code === 'Space') {
-    go = setInterval(startCar, 16)
+    numberOfSpacePresses += 1;
```

Comment:
Instead of doing this with a counter, why not use a boolean to indicate if the car is "running" or not?